commons/src/main/java/io/hyscale/commons/models/DockerCredHelper.java

```diff
@@ -60,11 +60,14 @@ public CredsStoreEntity get(String registryUrl) {
         try {
             CommandResult result = CommandExecutor.executeAndGetResults(command, registryUrl);
             ObjectMapper mapper = ObjectMapperFactory.jsonMapper();
-            if(result!=null && StringUtils.isNotBlank(result.getCommandOutput())&& result.getExitCode()!=1) {
-                CredsStoreEntity credsStore = mapper.readValue(result.getCommandOutput(), CredsStoreEntity.class);
+            if(result==null || StringUtils.isBlank(result.getCommandOutput()) || result.getExitCode()!=1) {
+                logger.debug("could'nt find credentials of registry {} in the credential store {}.",registryUrl,getHelperFunction());
+                return null;
```

Comment:
Every credhelper need not be a credstore
so just specify 
"Could not fetch credentials for registry {} from helper function {}"